lib/install.ps1

```diff
@@ -147,6 +147,116 @@ function do_dl($url, $to, $cookies) {
     }
 }
 
+function dl_with_cache_aria2($app, $version, $manifest, $architecture, $dir, $cookies = $null, $use_cache = $true, $check_hash = $true) {
+    $data = @{}
+    $urls = @(url $manifest $architecture)
+
+    # aria2 input file
+    $urlstxt = "$cachedir\$app.txt"
+    $urlstxt_content = ''
+    $has_downloads = $false
+
+    # aria2 options
+    $options = @(
```

Comment:
Users may want to tweak these defaults. Let’s add a config setting eventually.

Reply:
Should we use Scoops config settings or use a custom [aria2.conf](https://aria2.github.io/manual/en/html/aria2c.html#aria2-conf)? Should we allow aria2 to load the config file from `HOME` or disable it with [--no-conf](https://aria2.github.io/manual/en/html/aria2c.html#cmdoption-no-conf)?

Reply:
Good questions. I would say allow aria2 to use its default config file (`$HOME/.aria2/aria2.conf` or `$XDG_CONFIG_HOME/aria2/aria2.conf` if not found, `$XDG_CONFIG_HOME` defaults to `$HOME/.config`, see https://github.com/aria2/aria2/blob/e0a827ff98e665a118e2001dfe29b14847a33070/src/util.cc#L1808 )
In addition, perhaps we have a config setting, say `aria2_options`, to define scoop specific options?

Reply:
The desire behaviour must be scoop config taking precedence over aria2.conf. Maybe via script merge aria2.conf with scoop config and create new config file to get this behaviour. Things like number of pieces of downloads should be exposed on root of scoop config.

Thx